packages/wagtail/modules/components/Blocks/RichTextBlock/index.tsx

```
@@ -0,0 +1,14 @@
+'client side'
+
+import { RichTextBlockWrapper } from './styled'
+import { IRichTextBlockProps } from './types'
+
+const RichTextBlock = ({ value, WrapperProps }: IRichTextBlockProps) => (
+  <RichTextBlockWrapper
+    {...WrapperProps}
+    // eslint-disable-nex-line react/no-danger
+    dangerouslySetInnerHTML={{ __html: value }}
```

Biome

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)